lib/instrumentation/index.js

```diff
@@ -414,24 +414,26 @@ Instrumentation.prototype.addEndedSpan = function (span) {
     // if I have ended and I have something buffered, send that buffered thing
     if (span.getBufferedSpan()) {
       this._encodeAndSendSpan(span.getBufferedSpan())
+      span.setBufferedSpan(null)
```

Comment:
REVIEW NOTE: This clearing avoids the bug where we would encode and send a span *twice* -- the first time here, the second time in the next if-block.